src/item_loader.rs

```diff
@@ -132,5 +134,5 @@ fn load_items_from_file(file_path: &str) -> Result<Items, Box<dyn Error>> {
     let mut file_content = String::new();
     file.read_to_string(&mut file_content)?;
 
-    Ok(toml::from_str(&file_content)?)
+    Ok(toml::from_str(&file_content).expect("Toml format was not valid"))
```

Comment:
You sure about that?

Reply:
Oops, that one shouldn't have been committed 🙈. Reverted in af9696a